services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/configuration/ConfigurationSpec.groovy

```diff
@@ -79,4 +89,60 @@ class ConfigurationSpec extends HealthCheckSpecification {
                     new KildaConfigurationDto(flowEncapsulationType: defaultEncapsulationType))
         }
     }
+
+    def "System takes into account default multi table value while connecting a new switch"() {
+        assumeTrue("Multi table is not enabled in kilda configuration", useMultitable)
```

Comment:
Why? From the first look this test is designed to work correctly regardless of what config is currently set up